server/index.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const db = require('../database-mongodb/index.js');
+const Product = require('../database-mongodb/Product.js');
+const path = require('path');
+
+const app = express();
+const PORT = 3002;
+
+
+app.use(express.static(path.join(__dirname, '/../client/dist')));
+
+app.get('/api/product:id/sizechart', (req, res) => {
+  db.fetchFromDB()
```

Comment:
I'm a little confused by this line. Are you planning on implementing the `fetchFromDB()` function? Also, do you intend on invoking the function on the `db` object? I think you may have meant to invoke it on `Product` instead.